.github/workflows/pipeline-dito.yml

```diff
@@ -0,0 +1,38 @@
+name: PipelineDiTo
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+    paths:
+      - "packages/dito/**"
+      - ".github/workflows/**"
+      - "!.github/workflows/shared.yml"
+  pull_request:
+    branches: [main]
+    paths:
+      - "packages/dito/**"
+      - ".github/workflows/**"
+      - "!.github/workflows/shared.yml"
+
+jobs:
+  check-and-test:
+    uses: ./.github/workflows/check-and-test.yml
+    with:
+      flag: "--workspace=packages/dito"
```

Comment:
Why is the input the whole flag instead of just the package name (similar to the `build-and-deploy.yml` workflow? Are there current use cases where this is needed? Otherwise I would say YAGNI and KISS 🤪 and streamline aka. use only package name as input.

Reply:
Finde die Varianten alle nicht besonders schön: https://github.com/orgs/community/discussions/25725